examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,203 @@
     "tags": []
```

Comment:
<p>The bar plot about the averages is great. Can we also add a bar plot about the number of Rydberg atoms in each triangle? Namely, x = [0,1,2,3], y = #of triangles with x number of Rydbergs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
So for y, do we want to average over the number of shots? That would make y a fractional number, and the the label would be "Average number of triangles"